.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Required:
+------------
+[ ] I believe this isn't a duplicate topic  
+[ ] This report is not related to an adapter  
+
+For adapter related issues, please go to the appropriate repository - likely in
+the [OpenTimelineIO github organization](https://github.com/OpenTimelineIO/).
+For general questions and help please use the
+[Academy Software Foundation slack](https://slack.aswf.io/),
+[#opentimelineio](https://academysoftwarefdn.slack.com/messages/CMQ9J4BQC).
+
+## Select One:
+
+[ ] Build problem  
+[ ] Incorrect Functionality or bug  
+[ ] New feature or functionality  
+
+## Description
+<!-- Please make the description as precise as possible so we can best understand your suggestion or issue. -->
+
+
+# Optional
+-----------
+
+## Environment
+
+Operating System:
+Python version if appropriate:
```

Comment:
```suggestion
Operating System:  
Python version if appropriate:  
```
I think this should be the last of these newline suggestions. Sorry I didn't "catch'em'all" the first time around.